client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -0,0 +1,223 @@
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormLegend from 'components/forms/form-legend';
+import FormRadio from 'components/forms/form-radio';
+import FormCountrySelect from 'components/forms/form-country-select';
+import FormTextInput from 'components/forms/form-text-input';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { forDomainRegistrations as countriesListForDomainRegistrations } from 'lib/countries-list';
+
+const countriesList = countriesListForDomainRegistrations();
+
+class RegistrantExtraInfoForm extends React.PureComponent {
+	static propTypes = {
+		isVisible: PropTypes.bool,
+		onSubmit: PropTypes.func,
+		countriesList: PropTypes.object,
+		onStateChanged: PropTypes.func, // Just until we can reduxify the contact details
+	}
+
+	static defaultProps = {
+		isVisible: true,
+		isProbablyOrganization: false,
+		onSubmit: noop,
+		countriesList: countriesList,
+	}
+
+	constructor( props ) {
+		super( props );
+	}
+
+	componentWillMount() {
+		this.setState( {
+			registrantType: this.props.isProbablyOrganization
+				? 'organization' : 'individual',
+			countryOfBirth: 'AU',
+			dateOfBirth: '1978-01-01',
+			placeOfBirth: 'dummyCity',
+			postalCodeOfBirth: '12345',
+			registrantVatId: 'XX123456789',
+			sirenSiret: '123456789',
+			trademarkNumber: '123456789',
+		} );
+	}
+
+	getRelevantFields( state ) {
+		const { countryOfBirth, registrantType } = state;
+		const bornInFrance = countryOfBirth === 'FR';
+		const birthPlaceFields = [ 'placeOfBirth', 'postalCodeOfBirth' ];
+		const individualFields = [
+			'countryOfBirth',
+			'dateOfBirth',
+			...( bornInFrance ? birthPlaceFields : [] ),
+		];
+		const organizationalFields = [ 'registrantVatId', 'sirenSiret', 'trademarkNumber' ];
+		const conditionalFields = registrantType === 'individual'
+			? individualFields
+			: organizationalFields;
+		return [ 'registrantType', ...conditionalFields ];
+	}
+
+	handleChangeEvent = ( event ) => {
+		console.log( 'handleChangeEvent… event.target.name:', event.target.name, 'event.target.value:', event.target.value );
+		this.setState( { [ event.target.name ]: event.target.value } );
+	}
+
+	componentWillUpdate( _ /* nextProps */, nextState ) {
+		// This is pretty dirty :(
+		// The sooner we can get the contact details into the state, the better
+		this.props.onStateChanged && this.props.onStateChanged( nextState );
+	}
+
+	render = () => {
+		return (
+			<form>
+				<FormFieldset>
+					<FormLegend>
+						{ this.props.translate( "Who's this domain for?" ) }
+					</FormLegend>
+					<FormLabel>
+						<FormRadio value="individual"
+							checked={ 'individual' === this.state.registrantType }
+							name="registrantType"
+							onChange={ this.handleChangeEvent } />
+						<span>{ this.props.translate( 'An individual' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Individual' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).